### DISCUSSION TITLE
Connect to MongoDB backend of Unify Controller

### DISCUSSION THREAD

**Discussion by andreimoraru** [hassio-addons/addon-unifi#279] [General]
Hi,
I would need a way to connect to the MongoDB in order to attempt to clear the the MAC address of a Unifi AC Prod that fails to be adopted.
Could you please advise how I can attach to Docker instance or connect to it over network?

Thanks,

**Comment by numero-trey** [comment]
I need to connect to reset the password so I can migrate my controller. Something has been broken in the migration function for a long time and UI doesn't seem to want to fix it.

https://community.ui.com/questions/Lost-UniFi-Controller-Login-Password/61513566-c3b9-4128-9fa6-84034c083311
